starboard/build/config/BUILDCONFIG.gn

```diff
@@ -99,7 +99,13 @@ if (target_cpu == "x86" || target_cpu == "arm") {
 host_toolchain = "//starboard/build/toolchain/$host_os:$_host_toolchain_cpu"
 
 if (build_with_separate_cobalt_toolchain) {
-  cobalt_toolchain = "//starboard/build/toolchain:clang"
+  # TODO(b/294450490): decide a way to set cobalt_toolchain for modular builds decide for all platforms.
```

Comment:
I think there's a typo in this comment - should the second "decide" just be removed?